datafusion/core/src/logical_plan/mod.rs

```diff
@@ -53,6 +53,5 @@ pub use datafusion_expr::{
     reverse, right, round, rpad, rtrim, scalar_subquery, sha224, sha256, sha384, sha512,
     signum, sin, split_part, sqrt, starts_with, strpos, substr, sum, tan, to_hex,
     to_timestamp_micros, to_timestamp_millis, to_timestamp_seconds, translate, trim,
-    trunc, unalias, upper, when, Expr, ExprSchemable, Literal, Operator,
+    trunc, upper, when, Expr, ExprSchemable, Literal, Operator,
 };
-pub use datafusion_optimizer::expr_simplifier::SimplifyInfo;
```

Comment:
as the datafusion_optimizer is publically exported there is no reason to also re-export it here as well